codeworld-server/codeworld-server.cabal

```diff
@@ -39,7 +39,7 @@ Executable codeworld-server
     fast-logger,
     filelock,
     filepath,
-    haskell-src-exts < 1.21,
+    haskell-src-exts,
```

Comment:
Thanks!  I looked up the history, and this was originally added to keep hindent working, but CodeWorld now uses Ormolu instead of hindent, so dropping the restriction is fine.